src/platform/cmx/chal_cpu.h

```diff
@@ -0,0 +1,133 @@
+#ifndef CHAL_CPU_H
+#define CHAL_CPU_H
+
+#include <pgtbl.h>
+#include <thd.h>
+#include "stm32f7xx_hal.h"
+
+extern u32_t comp1_stack[1000];
+
+/* FIXME: I doubt these flags are really the same as the PGTBL_* macros */
+static inline u32_t
+chal_cpu_fault_errcode(struct pt_regs *r) { return r->r14_lr; }
+
+static inline u32_t
+chal_cpu_fault_ip(struct pt_regs *r) { return r->r15_pc; }
+
+extern unsigned int _c1_sdata;
```

Comment:
c1 means component 1, the booter component. These are now the variables defined in linker scripts.